src/anyio/to_thread.py

```diff
@@ -30,7 +30,7 @@ async def run_sync(
 
 
 async def run_sync_in_worker_thread(
-        func: Callable[..., T_Retval], *args, cancellable: bool = False,
+    func: Callable[..., T_Retval], *args: object, cancellable: bool = False,
```

Comment:
The change in indentation seems unwarranted; now it no longer aligns with the next line.